### PR TITLE
Truncate integral telemetry fields

### PR DESCRIPTION
**Pull request by konstantin-msft**
- Truncate integral telemetry fields.
- Remove broker timeouts (declared in 1p).

### CODE REVIEW

File: lib/msal-common/src/telemetry/performance/PerformanceClient.ts [hunk 1]
```diff
@@ -132,6 +139,13 @@ export abstract class PerformanceClient implements IPerformanceClient {
      */
     abstract setPreQueueTime(eventName: PerformanceEvents, correlationId?: string): void;
 
+    /**
+     * Get integral fields.
+     */
+    getIntFields(): ReadonlySet<string> {
```
Comment (tnorling) [inline lib/msal-common/src/telemetry/performance/PerformanceClient.ts:146]: Do we need this function or can we just directly use `IntFields`?

File: lib/msal-common/src/telemetry/performance/PerformanceClient.ts [hunk 2]
```diff
@@ -506,6 +520,7 @@ export abstract class PerformanceClient implements IPerformanceClient {
                     queuedCount: totalQueueCount,
                     incompleteSubsCount
                 };
+                this.truncateIntegralFields(finalEvent, this.getIntFields());
```
Comment (tnorling) [inline lib/msal-common/src/telemetry/performance/PerformanceClient.ts:524]: Then you don't need to pass anything in here...

File: lib/msal-common/src/telemetry/performance/PerformanceClient.ts (hunk 1 above)
Comment (konstantin-msft) [reply lib/msal-common/src/telemetry/performance/PerformanceClient.ts:146]: I have added it on purpose to allow overriding integral fields in 1p repo.

File: lib/msal-common/src/telemetry/performance/PerformanceClient.ts (hunk 2 above)
Comment (konstantin-msft) [reply lib/msal-common/src/telemetry/performance/PerformanceClient.ts:524]: Please see above comment.

File: lib/msal-common/src/telemetry/performance/PerformanceClient.ts (hunk 1 above)
Comment (peterzenz) [reply lib/msal-common/src/telemetry/performance/PerformanceClient.ts:146]: Maybe add a comment to the code itself so it's clear to the reader.

**Review by peterzenz — APPROVED** [review]

File: lib/msal-common/src/telemetry/performance/PerformanceClient.ts (hunk 1 above)
Comment (konstantin-msft) [reply lib/msal-common/src/telemetry/performance/PerformanceClient.ts:146]: Done

**Review by sameerag — APPROVED** [review]
Looks good.

**Pull request merged**